### DISCUSSION TITLE
Aide pour installation Gmail Kalliope

### DISCUSSION THREAD

**Discussion by versys650** [community/community#21439] [Other Feature Feedback, Questions, & Ideas]
<p>Bonsoir a tous,<br>
Jai installer Kalliope et tout fonctionne bien, j’essaye d’installer cet ressource mais j’ai une erreur et je debut sur kalliope pouvez vous m’aider  ci dessous le copie ecran de mon terminal</p>
<p>pi@raspberrypi:~ $ kalliope install --git-url <a href="https://github.com/kalliope-project/kalliope_neuron_gmail.git" class="inline-onebox" rel="noopener nofollow ugc">GitHub - kalliope-project/kalliope_neuron_gmail: A Kalliope Neuron to manage GMAIL</a><br>
Cloning repository…<br>
Checking repository…<br>
Resources folder for neuron installation not set in settings, cannot install.<br>
Traceback (most recent call last):<br>
File “/usr/local/bin/kalliope”, line 11, in <br>
load_entry_point(‘kalliope==0.7.1’, ‘console_scripts’, ‘kalliope’)()<br>
File “/usr/local/lib/python3.7/dist-packages/kalliope-0.7.1-py3.7.egg/kalliope/<strong>init</strong>.py”, line 109, in main<br>
res_manager.install()<br>
File “/usr/local/lib/python3.7/dist-packages/kalliope-0.7.1-py3.7.egg/kalliope/core/ResourcesManager.py”, line 92, in install<br>
raise ResourcesManagerException(“Invalid settings”)<br>
kalliope.core.ResourcesManager.ResourcesManagerException: Invalid settings</p>

**Comment by satana888** [comment, marked as answer]
<p>Bonjour,<br>
Dans ton fichier settings.yml tu dois lui donner les chemins pour les differents repertoires.</p>
<p><span class="hashtag">#resource_directory:</span><br>
neuron: “/var/tmp/resources/neurons”<br>
stt: “resources/stt”<br>
tts: “resources/tts”<br>
trigger: “resources/trigger”<br>
signal: “resources/signal”</p>
<p>Enlever dans un premier temps  les # , et si ça ne fonctionne pas encore, mettre tout les chemins en dur</p>
<p>J’espere que cela a pu t’aider   :slight_smile:</p>
<p>Bien à toi</p>

**Reply by UmeshMehta1** [reply]
okey

**Reply by UmeshMehta1** [reply]
great

**Comment by RicoFramboise** [comment]
Bonjour,

J'ai le même problème, j'ai voulu installer porcupine afin de remplacer snowboy et j'ai le même message

```
kalliope install --git-url https://github.com/kalliope-project/kalliope_trigger_porcupine.git
Cloning repository...
Checking repository...
Resources folder for trigger installation not set in settings, cannot install.
Traceback (most recent call last):
  File "/usr/local/bin/kalliope", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.9/dist-packages/kalliope/__init__.py", line 109, in main
    res_manager.install()
  File "/usr/local/lib/python3.9/dist-packages/kalliope/core/ResourcesManager.py", line 92, in install
    raise ResourcesManagerException("Invalid settings")
kalliope.core.ResourcesManager.ResourcesManagerException: Invalid settings
``` 

Cependant, j'ai le même message en faisant évoluer le settings dans le répertoire kalliope/kalliope
```
# ---------------------------
resource_directory:
  neuron: "/var/tmp/resources/neurons"
  stt: "stt"
  tts: "tts"
  trigger: "/home/pi/kalliope/kalliope/trigger"
  signal: "signal"
``` 

Merci pour votre aide

**Reply by RicoFramboise** [reply]
Bonjour,

J'ai réussi à installer la ressource, il fallait que je sois dans le dossier du kit de démarrage et là il prend correctement en compte le setting avec le chemin indiqué.
Après j'ai un autre problème mais ce n'est pas en lien avec ce sujet.
